### ISSUE TITLE
Repository (Git) paths with special characters sed error

### ISSUE THREAD

**Issue by ralfbiedert** [oh-my-fish/theme-bobthefish#78]
Changing to a repository with special characters in the path (e.g.) 

`#01 - My Git Repository` 

cause a `sed` error:

`sed: 1: "s#/Users/rb/Documents/T ...": bad flag in substitute command: '-'`.

The likely culprit is here:

```bash
function __bobthefish_project_pwd -S -a current_dir -d 'Print the working directory relative to project root'
  echo "$PWD" | sed -e "s#$current_dir##g" -e 's#^/##'
end
```

**Comment by bobthecow** [comment]
This should be fixed by 908a7b5365a937f295d9500402465de024a05244. Mind checking it out?

**Comment by ralfbiedert** [comment]
Yes. Pulled the latest master, seems to be fixed. Thanks!
